code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -231,7 +235,7 @@
 //It automatically updates damage overlays if necesary
 //It automatically updates health status
 /mob/living/carbon/human/heal_organ_damage(var/brute, var/burn)
-	var/list/obj/item/organ/external/parts = get_damaged_organs(brute,burn)
+	var/list/obj/item/organ/external/parts = get_damaged_organs(brute,burn,"BOTH")

```

Comment:
For clarification, this means you don't actually need to add "BOTH" to the proc call, as by default the `flag` variable is set to "BOTH".